packages/subgraph/schema.graphql

```diff
@@ -130,14 +130,20 @@ type StakeHistory @entity {
   createdAt: BigInt! # Timestamp at which was created
 }
 
-type Supporter @entity {
+type User @entity {
   id: ID!
   address: Bytes!
-  proposal: Proposal!
+  supports: [Supporter!]! @derivedFrom(field: "user")
+}
+
+type Supporter @entity {
+  id: ID!
+  user: User!
+  organization: Organization!
```

Comment:
Wonder if we want a link between the `User` entity and the organization. And we can consider members of the organization in the front-end to those that both exist in the subgraph as a `User` and in the present has at least a portion of the organization `GDN` token.

Reply:
I think that until it isn't strictly necessary to maintain a link between the organization to an array of users we can keep as is and just have a members counter at the organization entity level.
If in a future iteration we end up needing the list of users by organization then i think that makes sense yes. 

The members counter should take into account also the second condition you just described.